tardis/simulation/tests/test_simulation.py

```diff
@@ -7,8 +7,15 @@
 from tardis.simulation import Simulation
 
 
+''' I don't get the purpose of this test. It looks like it's trying to do unit
```

Comment:
comment's such as these should be added to the PR and not in the code. Code should only contain facts and not personal comments or questions.